crates/bevy_pbr/src/material.rs

```diff
@@ -186,6 +186,14 @@ pub trait SpecializedMaterial: Asset + RenderAsset {
     fn dynamic_uniform_indices(material: &<Self as RenderAsset>::PreparedAsset) -> &[u32] {
         &[]
     }
+
+    #[allow(unused_variables)]
+    #[inline]
+    /// Add a bias to the view depth of the mesh which can be used to force a specific render order
+    /// for meshes with equal depth, to avoid z-fighting.
+    fn depth_bias(material: &<Self as RenderAsset>::PreparedAsset) -> f32 {
```

Comment:
```suggestion
    #[inline]
    /// Add a bias to the view depth of the mesh which can be used to force a specific render order
    /// for meshes with equal depth, to avoid z-fighting.
    fn depth_bias(_material: &<Self as RenderAsset>::PreparedAsset) -> f32 {
```
This is not a Problem as trait decl and impl can have assign different Names to the sammeln binding.
Also the key fn in this trait also just preiendes a _

Reply:
If this is the convention elswhere in the codebase, then I don't have any problem keeping this like this.